src/main/java/core/basesyntax/model/User.java

```diff
@@ -19,7 +19,7 @@
     private Long id;
     private String username;
     @OneToMany(fetch = FetchType.EAGER)
-    @Cascade(value = CascadeType.SAVE_UPDATE)
+    @Cascade(value = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
Do we need an annotation for the update/merge in the condition?

Reply:
> When you are creating a new user you should use also create comments (save each new comment in the DB).
there is no point that we should update comments when the user is updated